src/main/java/Deadline.java

```diff
@@ -0,0 +1,17 @@
+public class Deadline extends Task{
```

Comment:
Comments are missing for the entire code. Good to have them for better readability.